moveit_setup_assistant/templates/moveit_config_pkg_template/launch/demo_gazebo.launch

```diff
@@ -1,21 +1,23 @@
+<?xml version="1.0"?>
 <launch>
-
-  <!-- specify the planning pipeline -->
-  <arg name="pipeline" default="ompl" />
-
   <!-- Gazebo specific options -->
-  <arg name="gazebo_gui" default="true"/>
-  <arg name="paused" default="false"/>
+  <arg name="world_name" default="worlds/empty.world"/>
+  <arg name="world_pose" default="-x 0 -y 0 -z 0 -R 0 -P 0 -Y 0"/>
+
+  <!-- Specify the planning pipeline -->
+  <arg name="pipeline" default="ompl"/>
 
-  <!-- launch the gazebo simulator and spawn the robot -->
-  <include file="$(dirname)/gazebo.launch" >
-    <arg name="paused" value="$(arg paused)"/>
-    <arg name="gazebo_gui" value="$(arg gazebo_gui)"/>
+  <!-- Launch the Gazebo simulator and spawn the robot -->
+  <include file="$(dirname)/gazebo.launch" pass_all_args="true" >
+    <arg name="world_name" value="$(arg world_name)"/>
+    <arg name="world_pose" value="$(arg world_pose)"/>
```

Comment:
As you removed the other arguments, these could be removed as well. We do have `pass_all_args=true`